src/database/operations/load/__test__/loadBundle.int.test.ts

```diff
@@ -1,27 +1,29 @@
-import { loadBundle } from '..';
-import { isApplicant } from '../../../../types';
-import { db } from '../../..';
+import { createOrganization, loadBundle } from '../../..';
+import { isOrganization } from '../../../../types';
 
 describe('loadBundle', () => {
 	it('Should return a bundle for the query', async () => {
-		await db.sql('applicants.insertOne', {
-			externalId: '12345',
-			optedIn: true,
+		await createOrganization({
+			employerIdentificationNumber: '11-111111',
```

Comment:
I believe this is meant to be a nine-digit EIN. (Only flagging once but I see it multiple times.)

```suggestion
			employerIdentificationNumber: '11-1111111',
```